deploy-os_servers.yml

```diff
@@ -486,6 +592,61 @@
         wait: true
         timeout: "{{ openstack_api_timeout }}"
 ##############################################################################
+# Configure IRODS servers from inventory using Openstack API.
+##############################################################################
+- name: Create IRODS catalogus servers.
+  hosts: irods
+  connection: local
+  gather_facts: false
+  vars:
+    #
+    # Disable Ansible's interpretor detection logic,
+    # which fails to use the interpretor from an activated virtual environment
+    # and hence fails to find the OpenStackSDK if it was installed in a virtual environment.
+    #
+    - ansible_python_interpreter: python
+  tasks:
+    - name: Create persistent data volume for irods server.
+      openstack.cloud.volume:
+        display_name: "{{ inventory_hostname }}-volume"
+        size: "{{ local_volume_size_irods }}"
+        state: present
+        availability_zone: "{{ storage_availability_zone | default('nova') }}"
+        wait: true
+        timeout: "{{ openstack_api_timeout }}"
+    - name: Create irods server.
+      openstack.cloud.server:
+        state: present
+        name: "{{ inventory_hostname }}"
+        image: "{{ cloud_image }}"
+        flavor: "{{ flavor_irods }}"
+        security_groups: "{{ slurm_cluster_name }}_irods"
+        auto_floating_ip: false
+        nics:
+          - net-name: "{{ network_private_management_id }}"
+          - net-name: "{{ network_private_management_id_13 }}"
+        userdata: |
+          #cloud-config
+          password: "{{ cloud_console_pass }}"
+          chpasswd: { expire: False }
+          #
+          # Add each entry to ~/.ssh/authorized_keys for the configured user
+          # or the first user defined in the user definition directive.
+          #
+          ssh_authorized_keys:
+          {% for public_key in public_keys_of_local_admins %}  - {{ public_key }}
+          {% endfor %}
+        availability_zone: "{{ availability_zone }}"
+        wait: true
+        timeout: "{{ openstack_api_timeout }}"
+      register: irods_vm
+    - name: Make sure irods nodes are started.
+      openstack.cloud.server_action:
+        action: start
+        server: "{{ inventory_hostname }}"
+        wait: true
+        timeout: "{{ openstack_api_timeout }}
```

Comment:
`"` not closed at the end of the line results in failure to parse this file -> CircleCI complains.